mimic/canned_responses/maas_alarm_examples.py

```diff
@@ -294,3 +294,20 @@ def alarm_examples():
             "type": "whole number (may be zero padded)"
         }]
     }]
+
+
+def alarm_examples_template(metric, crit_threshold, warn_threshold):
+    """
+    There's actually a POST version of this to ask for a specific alarm example
+    """
+    alarm_templates = {}
+    alarm_templates['agent.cpu_usage_average'] = {
+        "bound_criteria": "if (metric['usage_average'] > CRIT_THRESHOLD) {\nreturn new AlarmStatus" +
+        "(CRITICAL,'CPU usage is #{usage_average}%, above your critical threshold of CRIT_THRESHOLD%')" +
+        ";\n}\n\nif (metric['usage_average'] > WARN_THRESHOLD) {\nreturn new AlarmStatus(WARNING, 'CPU" +
+        "usage is #{usage_average}%, above warning threshold of WARN_THRESHOLD%');\n}\nreturn new " +
+        "AlarmStatus(OK, 'CPU usage is #{usage_average}%, below your warning threshold of " +
+        "WARN_THRESHOLD%');\n"
+        .replace("CRIT_THRESHOLD", crit_threshold).replace("WARN_THRESHOLD", warn_threshold)
+    }
+    return alarm_templates[metric]
```

Comment:
Is the intention for this function to raise a `KeyError` if passed anything other than `agent.cpu_usage_average` for `metric`?